.dvc/.gitignore

```diff
@@ -0,0 +1,9 @@
+/config.local
+/updater
+/state-journal
```

Comment:
interesting (the leading `/`). So these are definitely only in the root folder where the `.gitignore` is located? Cool

Reply:
You are correct